.github/workflows/build.yaml

```diff
@@ -111,13 +111,13 @@ jobs:
 
       # Note that this approach to cloning repos obviates any modification to
       # the CMake variables in get_cumlprims_mg.cmake since CMake will just use
-      # the clone as is.
+      # the clone as-is.
       extra-repo: rapidsai/cugraph-ops
-      extra-repo-sha: branch-23.08
+      extra-repo-sha: branch-23.06
```

Comment:
Looks like this switched to `branch-23.06` instead of `branch-23.08`. Did this merge a different branch than `branch-23.08`? Is the latest `branch-23.08` fetched locally?